Modules/Scripted/SampleData/SampleData.py

```diff
@@ -415,13 +415,20 @@ def registerCustomSampleDataSource(category='Custom',
       customDownloader=customDownloader,
       ))
 
+    
+  @staticmethod
+  def CustomDownloadTest(source):
```

Comment:
Compare source parameter is the registered source and raise an `AssertError` if not the expected one.